src/network/simulated.rs

```diff
@@ -265,12 +268,13 @@ mod tests {
         let mut rng = rand::rng();
         let sk = SecretKey::new(&mut rng);
         let mut shreds = Vec::new();
-        for i in 0..10_000 {
+        let final_slice_index = SliceIndex::new_unchecked(9_999);
```

Comment:
@qkniep: I just realized that this test will fail now because we are capping max slice index at 1024.  I suppose I can introduce a way to create a slice index that does not assert that it is < 1024 and use that here.  Or should we consider rewriting the test to use multiple slots, or any other ideas?